src/metrics/exit.rs

```diff
@@ -121,3 +140,27 @@ impl Exit for JavaCode {}
 impl Exit for GoCode {}
 impl Exit for CssCode {}
 impl Exit for HtmlCode {}
+
+#[cfg(test)]
+mod tests {
+    use std::path::PathBuf;
+
+    use super::*;
+
+    #[test]
+    fn test_nested_functions_exit() {
+        check_metrics!(
+            "def f(a, b):
+                 def foo(a):
+                     if a:
+                         return 1
+                 bar = lambda a: lambda b: b or True or True
+                 return bar(foo(a))(a)",
+            "foo.py",
+            PythonParser,
+            nexits,
+            [(exit, 2, usize)],
```

Comment:
Shouldn't there be 4 exits here in total (one for `f`, one for `foo`, one for each lambda)?

Reply:
We don't consider closures exit points for now. Perhaps we should add them in another PR

Reply:
Can you file an issue about that, so we don't forget?

Reply:
https://github.com/mozilla/rust-code-analysis/issues/341